docs/modules/globalfee.md

```diff
@@ -29,41 +31,40 @@ A global fees list must meet the following properties:
 - fees have to be alphabetically sorted by denom; 
```

Comment:
Additional requirements for the fees include:
- fees have to be alphabetically sorted by denom
- fees must have non-negative amount, with a valid and unique denom (i.e. no duplicate denoms are allowed)